models/spring-ai-watsonx-ai/src/main/java/org/springframework/ai/watsonx/WatsonxAiChatOptions.java

```diff
@@ -99,7 +103,7 @@ public class WatsonxAiChatOptions implements ChatOptions {
      * (e.g., 1.8) will penalize repetitions more strongly, while a lower value (e.g.,
      * 1.1) will be more lenient. (Default: 1.0)
      */
-    @JsonProperty("repetition_penalty") private Float repetitionPenalty;
+    @JsonProperty("repetition_penalty") private Float repetitionPenalty = 1.0f;
```

Comment:
Do not set default values here but your default instance in https://github.com/spring-projects/spring-ai/blob/9236913696298bd42838bab0b72416ab413bf8d3/models/spring-ai-watsonx-ai/src/main/java/org/springframework/ai/watsonx/WatsonxAiChatClient.java#L57
or https://github.com/spring-projects/spring-ai/blob/9236913696298bd42838bab0b72416ab413bf8d3/spring-ai-spring-boot-autoconfigure/src/main/java/org/springframework/ai/autoconfigure/watsonxai/WatsonxAiChatProperties.java#L42